docs/openapi.yaml

```
@@ -1341,4 +1446,242 @@ components:
+        IsSuperAdmin:
+          type: boolean
+          example: true
+        DisplayName:
+          type: string
+          example: ""
+        Error:
+          type: integer
+          example: 0
+        AddRequest:
+          type: string
+          example: null
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)